frontend/src/pages/modelRegistry/ModelRegistryCoreLoader.tsx

```diff
@@ -52,16 +56,37 @@ const ModelRegistryCoreLoader: React.FC<ModelRegistryCoreLoaderProps> =
       renderStateProps = {
         empty: true,
         emptyStatePage: (
-          // TODO: Replace this with a component for empty registries once we have the designs
           <EmptyModelRegistryState
-            title="No model registries found"
-            description="No model registries found in the cluster. Configure a new one before registering models."
-            primaryActionText="Configure model registry"
-            primaryActionOnClick={() => {
-              // TODO: Add primary action
-            }}
+            testid="empty-model-registries-state"
+            title="Request access to model registries"
+            description="To request a new model registry, or to request permission to access an existing model registry, contact your administrator."
+            headerIcon={() => (
+              <img src={typedEmptyImage(ProjectObjectType.registeredModels)} alt="" />
+            )}
+            customAction={
+              <Popover
+                showClose
+                position="bottom"
+                headerContent="Your administrator might be:"
+                bodyContent={
+                  <List>
+                    <ListItem>
+                      The person who gave you your username, or who helped you to log in for the
+                      first time
+                    </ListItem>
+                    <ListItem>Someone in your IT department or help desk</ListItem>
+                    <ListItem>A project manager or developer</ListItem>
+                  </List>
+                }
+              >
+                <Button variant="link" icon={<OutlinedQuestionCircleIcon />} isInline>
```

Comment:
The `isInline` property should only be used when the button is placed inline with other texts thus isn't needed here.

Reply:
Without `isInline`, the underline which you explicitly asked for no longer shows up on hover of the link. So which would you prefer? No underline on hover or remove `isInline` because it has no text next to it?

Reply:
If I'm understanding this earlier comment correctly:

> The link button variant should be used, and when applied correctly, the link will not have an underline while hovering.

I think @yih-wang was asking for no underline on hover, which would be consistent with removing `isInline` here

Reply:
Yes, I meant the button shouldn't display an underline when hovering.

Reply:
Your design had the underline, but it appears it doesn't any longer. I've now removed the property.

Reply:
Right it appears to be a bug (incorrect style) in the Figma library, so I then fixed that. Thanks for the updates!